testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/BasicOpsTestCase.java

```diff
@@ -72,26 +72,24 @@ public void testLs() throws Exception {
     public void testReadAttribute() throws Exception {
         CLIWrapper cli = new CLIWrapper(true);
         {
-            cli.sendLine("read-attribute namespaces");
-            String output = cli.readOutput();
-            assertFalse(output.contains("Unknown attribute 'namespaces'"));
+            assertTrue("Failed to read attribute: " + cli.readOutput(),
+                    cli.sendLine("read-attribute namespaces", false));
         }
         {
-            cli.sendLine("read-attribute --node=subsystem=request-controller namespaces", true);
+            assertFalse("Expected command to fail on invalid node",
+                    cli.sendLine("read-attribute --node=subsystem=request-controller namespaces", true));
             String output = cli.readOutput();
-            assertTrue(output.contains("Unknown attribute 'namespaces'"));
+            assertTrue(output.matches("WFLYCTL0201.*namespaces.*"));
         }
 
         {
-            cli.sendLine("read-attribute active-requests --node=subsystem=request-controller");
-            String output = cli.readOutput();
-            assertFalse(output.contains("Unknown attribute 'active-requests'"));
+            assertTrue("Failed to read attribute: " + cli.readOutput(),
+                    cli.sendLine("read-attribute active-requests --node=subsystem=request-controller", false));
         }
 
         {
-            cli.sendLine("read-attribute --node=subsystem=request-controller active-requests");
-            String output = cli.readOutput();
-            assertFalse(output.contains("Unknown attribute 'active-requests'"));
+            assertTrue("Failed to read attribute: " + cli.readOutput(),
+                    cli.sendLine("read-attribute --node=subsystem=request-controller active-requests", false));
```

Comment:
The way I see the test, it seems that true should be passed to sendLine method call in order to ignore error and have the return value be true if no error, false is error occurred.